pkg/iac/terraform/state/backend/azureblob_reader.go

```diff
@@ -0,0 +1,69 @@
+package backend
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+	"github.com/pkg/errors"
+	"github.com/snyk/driftctl/pkg/helpers/azure"
+)
+
+const BackendKeyAzureBlob = "azureblob"
```

Comment:
Not sure if I should follow [terraform naming](https://www.terraform.io/language/settings/backends/azurerm) here, `azurerm` does not resonates to me.

WDYT ?

Reply:
For users that use this kind of backend, it would make more sense use the same naming.

Reply:
Renamed ✅